### PR TITLE
bump: akka-grpc 2.3.4

### PR DESCRIPTION
**Pull request by sebastian-alfers**

### CODE REVIEW

File: docs-source/docs/modules/how-to/examples/shopping-cart-service-java/pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <akka-grpc-maven-plugin.version>2.3.3</akka-grpc-maven-plugin.version>
+        <akka-grpc-maven-plugin.version>2.3.4</akka-grpc-maven-plugin.version>
```
Comment (patriknw) [inline docs-source/docs/modules/how-to/examples/shopping-cart-service-java/pom.xml:20]: There is also a `akka-grpc-runtime` in the maven builds. Before completing this PR we would have to release a new  akka-dependencies bom with the new version
https://github.com/lightbend/akka-dependencies

Comment (sebastian-alfers) [reply docs-source/docs/modules/how-to/examples/shopping-cart-service-java/pom.xml:20]: So better revert this for now?

Comment (patriknw) [reply docs-source/docs/modules/how-to/examples/shopping-cart-service-java/pom.xml:20]: we can leave it as draft until we have the next release of the bom

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**